packages/suite-desktop/src/support/Router.tsx

```diff
@@ -47,6 +48,7 @@ const components: { [key: string]: React.ComponentType<any> } = {
     'wallet-send': WalletSend,
     'wallet-staking': WalletStaking,
     'wallet-sign-verify': WalletSignVerify,
+    'wallet-coinjoin': WalletCoinjoin,
```

Comment:
Perhaps like that? Since it's the setup page.

```suggestion
    'wallet-coinjoin-setup': WalletCoinjoin,
```

Reply:
I renamed it to `wallet-anonymize` which is both short and accurate in my opinion. fcbc1400acbd07df4d7541c35a689a99b9babf0d